docs/source/install.rst

```diff
@@ -84,6 +85,96 @@ Raspberry Pi OS
     sudo curl -fL http://docs.luxonis.com/_static/install_dependencies.sh | bash
 
 
+Jetson Nano
+***********
+
+To install DepthAI on Jetson Nano, perform the following steps, after completing a fresh install and setup. On the first log in, 
+**do not** immediately run updates.
+
+Go to the *Software* (App Store) and delete the apps or software that you probably will not use. Then open a terminal window and run the following commands:
+  
+  .. code-block:: bash
+
+    sudo apt update && sudo apt upgrade
+    sudo reboot now
+
+Change the size of your SWAP. These instructions come from the `Getting Started with AI on Jetson Nano <https://developer.nvidia.com/embedded/learn/jetson-ai-certification-programs>`__ from nvidia:
+
+  .. code-block:: bash
+
+    # Disable ZRAM:
+    sudo systemctl disable nvzramconfig
+    # Create 4GB swap file
+    sudo fallocate -l 4G /mnt/4GB.swap
+    sudo chmod 600 /mnt/4GB.swap
+    sudo mkswap /mnt/4GB.swap
+
+If you have an issue with the final command, you can try the following:
+
+    .. code-block:: bash
+
+      sudo vi /etc/fstab
+
+      # Add this line at the bottom of the file
+      /mnt/4GB.swap swap swap defaults 0 0
+
+      # Reboot 
+      sudo reboot now
+
+The next step is to install :code:`pip` and :code:`python3`:
+
+  .. code-block:: bash
+  
+    sudo -H apt install -y python3-pip
+
+After that, install and set up virtual environment:
+
+  .. code-block:: bash
+
+    sudo -H pip3 install virtualenv virtualenvwrapper
+
+Add following lines to the bash script:
+
+  .. code-block:: bash
+
+    sudo vi ~/.bashrc
+
+    # Virtual Env Wrapper Configuration
+    export WORKON_HOME=$HOME/.virtualenvs
+    export VIRTUALENVWRAPPER_PYTHON=/usr/bin/python3
+    source /usr/local/bin/virtualenvwrapper.sh
+
+Save and reload the script by running the command :code:`source ~/.bashrc`. Then create a virtual environment (in this example it's called :code:`depthAI`).
+  
+  .. code-block:: bash
+
+    mkvirtualenv depthAI -p python3
+
+
+**Note!** Before installing :code:`depthai`, make sure you're in the virtual environment.
+
+  .. code-block:: bash
+
+    sudo apt install curl
+
+    #Download and install the dependency package
+    sudowget-qO- http://docs.luxonis.com/_static/install_dependencies.sh | bash
+
+    #Clone github repository
+    git clone https://github.com/luxonis/depthai-python.git
+    cd depthai-python
+
+Last step is to edit :code:`.bashrc` with the line:
+
+  .. code-block:: bash
+
+    echo “export OPENBLAS_CORETYPE=AMRV8” >> ~/.bashrc
+
+Navigate to the folder with :code:`depthai` examples and run it.
```

Comment:
This needs to be changed, we need to specify we need to go to examples folder, run `python install_requirements.py` and then `python 01_rgb_preview.py`